libdevice/spirv_vars.h

```diff
@@ -0,0 +1,50 @@
+//==---------- spirv_vars.h --- SPIRV variables --------------*- C++ -*-----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __LIBDEVICE_SPIRV_VARS_H
+#define __LIBDEVICE_SPIRV_VARS_H
+
+#include "device.h"
+
+#if __SPIR_TARGET_ONLY__
+
+#ifdef CL_SYCL_LANGUAGE_VERSION
+// SYCL provides all the needed definitions.
+#include "CL/__spirv/spirv_vars.hpp"
+#else // CL_SYCL_LANGUAGE_VERSION
+
```

Comment:
Thanks.  I have decided to remove this, since this will make it easier to introduce issues for non-SYCL compilations, which won't be noticed in this repo.